cloudify_agent/tests/api/pm/test_base.py

```diff
@@ -32,7 +32,9 @@ def setUp(self):
         self.daemon = Daemon(
             manager_ip='manager_ip',
             queue='queue',
-            name='name'
+            name='name',
+            broker_user='guest',
```

Comment:
True, but this means that if someone changes that method's signature then this test will fail (rightly), so probably worth leaving in?